src/components/AvatarGroup/readme.md

```diff
@@ -0,0 +1,460 @@
+##### photo-url
+
+```js
+import React from 'react';
+import { AvatarGroup } from 'react-rainbow-components';
+
+const avatars = [
+    {
+        src:"images/user/user1.jpg",
+        assistiveText:"Jose Leandro",
+        title:"Jose Leandro",
+    },
+    {
+        src:"images/user/user2.jpg",
+        assistiveText:"Tahimi Leon",
+        title:"Tahimi Leon",
+    },
+    {
+        src:"images/user/user3.jpg",
+        assistiveText:"Carlos Miguel",
+        title:"Carlos Miguel",
+    },
+    {
+        src:"images/user/user4.jpg",
+        assistiveText:"Jane Doe",
+        title:"Jane Doe",
+    },
+];
+
+<div className="rainbow-p-vertical_large rainbow-p-left_medium rainbow-flex_column rainbow-align_center">
+    <AvatarGroup
+        className="rainbow-m-horizontal_medium"
+        avatars={avatars}
+        maxAvatars={4}
+    />
+    <AvatarGroup
+        className="rainbow-m-horizontal_medium"
+        avatars={avatars}
+        maxAvatars={3}
+        showCounter={true}
+    />
+</div>
```

Comment:
I think that in the first example we can add all the avatars types `url`, `initials` and `icon` instead create an example for each avatar type